packages/chart/src/components/EditorPanel/EditorPanel.tsx

```diff
@@ -1444,19 +1444,21 @@ const EditorPanel = () => {
                         label='Series Type'
                         initial='Select'
                         updateField={updateField}
-                        options={['Line','dashed-sm', 'dashed-md', 'dashed-lg']}
-                      />
-                    )}
-                    {config.dynamicSeries && config.visualizationType === 'Line' && config.dynamicSeriesType === 'Line' && (
-                      <Select
-                        fieldName='dynamicSeriesLineType'
-                        value={config.dynamicSeriesLineType ? config.dynamicSeriesLineType : 'curveLinear'}
-                        label='Line Type'
-                        initial='Select'
-                        updateField={updateField}
-                        options={Object.keys(approvedCurveTypes).map(curveName => approvedCurveTypes[curveName])}
+                        options={['Line', 'dashed-sm', 'dashed-md', 'dashed-lg']}
```

Comment:
I'm not seeing any changes in this file, if none can you remove from your commit history?